client/src/components/innerComponents/homeCategories/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { useRef } from 'react';
+import clamp from 'lodash-es/clamp';
+import swap from 'lodash-move';
+import { useGesture } from 'react-use-gesture';
+import { useSprings, animated, interpolate } from 'react-spring';
+
+const fn = (order, down, originalIndex, curIndex, y) => index =>
+  down && index === originalIndex
+    ? { y: curIndex * 100 + y, scale: 1.1, zIndex: '1', shadow: 15, immediate: n => n === 'y' || n === 'zIndex'  }
+    : { y: order.indexOf(index) * 100, scale: 1, zIndex: '0', shadow: 1, immediate: false }
+    
+    export default function HomeCategory() {
+      let items ='Lorem ipsum dolor sit'.split(' ')
+    const order = useRef(items.map((_, index) => index)) // Store indicies as a local ref, this represents the item order
+  
+    const [springs, setSprings] = useSprings(items.length, fn(order.current)) // Create springs, each corresponds to an item, controlling its transform, scale, etc.
+ 
+    const bind = useGesture(({ args: [originalIndex], down, delta: [, y] }) => {
+      console.log(order.current.indexOf(originalIndex))
+      const curIndex = order.current.indexOf(originalIndex)
+      console.log(curIndex)
```

Comment:
I am trying to be better about this too, but let's try to comment or remove logs when pushing to Main. Agree or disagree? :)

Reply:
I agree for the most part -- left those in specifically bc that code is not functioning still and I started an issue about it. I will double check the other pages for unnecessary console.logs(). I think there is one other that I will leave in since it shows the object that will eventually be sent to the backend.